src/ProcessService/include/ProcessService/CpuTime.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef PROCESS_SERVICE_CPU_TIME_H_
+#define PROCESS_SERVICE_CPU_TIME_H_
+
+namespace orbit_process_service_internal {
```

Comment:
I take it this is not supposed to be used by other modules, in which case maybe move it to src/ProcessService so that is not visible by design.

I am not sure about introducing another namespace, I think in this case we do not need it.

Reply:
`ProcessServiceImpl` class has a `ProcessList` field, which in turns has a field using `Process`, which in turns uses the `struct`s in this file.

Reply:
ok, I see. internal it is then,